www/index.html

```diff
@@ -0,0 +1,451 @@
+                </div>
+
+        </div>
+
+        <hr class="featurette-divider">
+        <div id="contact" class="container pad">
+            <h2 class="text-center pad">Who We Are</h2>
+            <div class="row">
+                <!-- bootstrap has 12 columns, so each person gets part of that 12 cols -->
+
+                <div class="col-lg-3">
+                    <img class="img-circle center-block" src="media/people/alexpap.jpg" alt="Alexandra Papoutsaki" height="140">
+                    <h4 class="text-center"><a href="http://www.cs.pomona.edu/~apapoutsaki/">Alexandra Papoutsaki</a></h4>
+                    <p>Assistant Professor of Computer Science at Pomona College.</p>
+                </div>
+                <div class="col-lg-2">
+                    <img class="img-circle center-block" src="media/people/jim.jpg" alt="James Laskey" height="140">
+                    <h4 class="text-center">James Laskey</h4>
+                    <p>Software Enginner at Google. Brown '16.</p>
+                </div>
+                <div class="col-lg-2">
+                    <img class="img-circle center-block" src="media/people/aaron.jpg" alt="Aaron Gokaslan" height="140">
+                    <h4 class="text-center">Aaron Gokaslan</h4>
+                    <p>Undergraduate student at Brown University.</p>
+                </div>
+                <div class="col-lg-3">
+                    <img class="img-circle center-block" src="media/people/yuze.jpg" alt="Yuze He" height="140">
+                    <h4 class="text-center">Yuze He</h4>
+                    <p>Masters student at Brown University.</p>
+                </div>
+                <div class="col-lg-2">
+                    <img class="img-circle center-block" src="media/people/jeff.jpg" alt="Jeff Huang" height="140">
+                    <h4 class="text-center"><a href="http://jeffhuang.com">Jeff Huang</a></h4>
+                    <p>Assistant Professor of Computer Science at Brown University.</p>
+                </div>
+            </div>
+            <div class="pad">
+                <h3>Other Collaborators</h3>
+                <ul style="margin-top: 20px">
+                    <li><a href="https://cs.brown.edu/~ptunnell/">Preston Tunnell Wilson</a> - PhD student at Brown University</li>
+                    <li><a href="http://www.cc.gatech.edu/~psangklo/">Patsorn Sangkloy</a> - PhD student at Georgia Institute of Technology</li>
+                    <li><a href="http://cs.brown.edu/~nediyana">Nediyana Daskalova</a> - PhD student at Brown University</li>
+                    <li><a href="http://www.cc.gatech.edu/~hays/">James Hays</a> - Associate Professor at Georgia Institute of Technology</li>
+                    <li><a href="https://www.linkedin.com/in/idakdesmet/">Ida De Smet</a> - Software engineering student at the University of Auckland</li>
+                    <li>Elizabeth Stevenson - Software engineering student at the University of Auckland</li>
```

Comment:
@idaknow  would you mind adding the link to my linkedin account here too? if that's not a hassle!
https://www.linkedin.com/in/elizabeth-stevenson-9b47a7125/

Reply:
@Jack-Wong94 do you have a linkedin account?